plugin/proto/metadata.go

```diff
@@ -13,30 +13,19 @@
 
 package proto
 
+import "github.com/notaryproject/notation-plugin-framework-go/plugin"
+
 // GetMetadataRequest contains the parameters passed in a get-plugin-metadata
 // request.
-type GetMetadataRequest struct {
-	PluginConfig map[string]string `json:"pluginConfig,omitempty"`
-}
-
-func (GetMetadataRequest) Command() Command {
-	return CommandGetMetadata
-}
+type GetMetadataRequest = plugin.GetMetadataRequest
 
 // GetMetadataResponse provided by the plugin.
-type GetMetadataResponse struct {
-	Name                      string       `json:"name"`
-	Description               string       `json:"description"`
-	Version                   string       `json:"version"`
-	URL                       string       `json:"url"`
-	SupportedContractVersions []string     `json:"supportedContractVersions"`
-	Capabilities              []Capability `json:"capabilities"`
-}
+type GetMetadataResponse = plugin.GetMetadataResponse
 
 // HasCapability return true if the metadata states that the
 // capability is supported.
 // Returns true if capability is empty.
-func (resp *GetMetadataResponse) HasCapability(capability Capability) bool {
```

Comment:
Instead of changing this function (which is a breaking change), we should move this function to `notation-plugin-framework-go/plugin/metadata.go`.